### ISSUE TITLE
Include payment transaction details in reglist / export

### ISSUE THREAD

**Issue by ThiefMaster** [indico/indico#3029]
This is useful e.g. for managers to create invoices that need to mention extra payment fees. It should be up to each payment plugin what to include in the details.

In the reglist column we can just use the rendering of the transaction details we also use when viewing a registration. For the CSV/Excel export we'd need a new method in the payment plugin that returns the data to include (ideally spread over multiple columns, depending on the payment plugin - if we do that we need to take care of cases where a file contains registrations with transactions from different payment providers!)

related snow ticket: INC1419357

**Comment by pferreir** [comment]
Do you mean just exporting billable fields as separate columns with the corresponding price?

**Comment by ThiefMaster** [comment]
No, as far as I understood the ticket this is about the payment transaction details (to get e.g. the credit card fee)

**Comment by pferreir** [comment]
Ah, right. Well, that seems reasonable.

**Comment by Problemkerze** [comment]
exporting billable fields as separate columns as pferreir mentioned would also be a nice feature, especially for a csv-export-option.
